client-keystore/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "substrate-client-keystore"
-version = "0.8.0"
+version = "0.8.1"
```

Comment:
I believe keystore does have breaking changes, with functions renaming and so on